### PR TITLE
added black border, add header to login, linked login success page, r…

### PR DESCRIPTION
**Pull request by agamble33**
…esized login, added hyperlink to splash page, formated signup

### CODE REVIEW

File: chucks-pick3-web/assets/css/app-instructions-style-sheet.css [hunk 1]
```diff
@@ -11,23 +11,51 @@
...
     display: block;
-    width: 60%;
-    height: 500px;
+    width: 100%;
+    height: 300px;
+    padding: 0;
+    margin: 0 auto;
+    position: center;
+    float: left;
+    border: 5px solid red;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/app-instructions-style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/app-instructions-style-sheet.css [hunk 2]
```diff
@@ -11,23 +11,51 @@
...
+
+.what-you-need {
+    display: block;
+    width: 100%;
+    height: 300px;
+    padding: 0;
+    margin: 0 auto;
+    position: center;
+    float: left;
+    border: 5px solid saddlebrown;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/app-instructions-style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/app-instructions-style-sheet.css [hunk 3]
```diff
@@ -11,23 +11,51 @@
...
+.interface {
+
+    display: block;
+    width: 100%;
+    height: 300px;
     padding: 0;
     margin: 0 auto;
     position: center;
     float: left;
+    border: 5px solid pink;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/app-instructions-style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/app-instructions-style-sheet.css [hunk 4]
```diff
@@ -11,23 +11,51 @@
...
     display: block;
-    width: 60%;
-    height: 500px;
+    width: 100%;
+    height: 300px;
     padding: 0px;
     margin: 0 auto;
     position: center;
     float: left;
+    border: 5px solid green;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/app-instructions-style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/global-style-sheet.css [hunk 5]
```diff
@@ -55,12 +55,15 @@ p {
...
 .body-container {
+
     display: block;
     background-color: #ffffff;
     width: 60%;
     height: 90%;
     padding: 0;
     margin: 0 auto;
+    margin-top: 20px;
+    border: 5px solid blue;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/global-style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/global-style-sheet.css [hunk 6]
```diff
@@ -153,13 +165,17 @@ h1 {
     height: 100%;
     display: block;
     float: left;
+    margin-top: 20px;
+    border: 5px solid yellow;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/global-style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/global-style-sheet.css [hunk 7]
```diff
@@ -153,13 +165,17 @@ h1 {
...
 
 }
 .demo-section {
     width: 100%;
     height: 100%;
     display: block;
     float: left;
+    margin-top: 20px;
+    border: 5px solid darkorchid;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/global-style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/style-sheet.css [hunk 8]
```diff
@@ -136,22 +135,21 @@ p {
...
-    background-color: #fafafa;
     width: 100%;
-    height: 500px;
-    padding: 0px;
+    height: 300px;
+    padding: 0;
     margin: 0 auto;
     position: center;
     float: left;
+    border: 5px solid pink;
```
Comment (ocampd1) [inline chucks-pick3-web/assets/css/style-sheet.css]: Could the borders be removed?

File: chucks-pick3-web/assets/css/app-instructions-style-sheet.css (hunk 1 above)
Comment (agamble33) [reply chucks-pick3-web/assets/css/app-instructions-style-sheet.css]: removed

File: chucks-pick3-web/assets/css/app-instructions-style-sheet.css (hunk 2 above)
Comment (agamble33) [reply chucks-pick3-web/assets/css/app-instructions-style-sheet.css]: done

**Pull request closed without merging**